apertium-tur-uzb.tur-uzb.dix

```diff
@@ -1,4716 +1,4198 @@
+<e>       <p><l>çare<s n="n"/></l>                     <r>chora<s n="n"/></r></p></e>
+<e>       <p><l>çarpma<s n="n"/></l>                   <r>koʻpaytirish<s n="n"/></r></p></e>
+<e>       <p><l>çavuş<s n="n"/></l>                    <r>serjant<s n="n"/></r></p></e>
+<e>       <p><l>çay<s n="n"/></l>                      <r>choy<s n="n"/></r></p></e>
+<e>       <p><l>çeviri<s n="n"/></l>                   <r>tarjima<s n="n"/></r></p></e>
+<e>       <p><l>çevre<s n="n"/></l>                    <r>atrof<s n="n"/></r></p></e>
+<e>       <p><l>çift<s n="n"/></l>                     <r>juft<s n="n"/></r></p></e>
+<e>       <p><l>çifte<s n="n"/></l>                    <r>qoʻsh<s n="n"/></r></p></e>
+<e>       <p><l>Çinli<s n="n"/></l>                    <r>chiniy<s n="n"/></r></p></e>
+<e>       <p><l>çırak<s n="n"/></l>                    <r>shogird<s n="n"/></r></p></e>
+<e>       <p><l>çizgi<s n="n"/></l>                    <r>liniya<s n="n"/></r></p></e>
+<e>       <p><l>çiçek<s n="n"/></l>                    <r>chehcak<s n="n"/></r></p></e>
+<e>       <p><l>çiçek<s n="n"/></l>                    <r>chechak<s n="n"/></r></p></e>
+<e>       <p><l>çocuk<s n="n"/></l>                    <r>bola<s n="n"/></r></p></e>
+<e>       <p><l>çoğunluk<s n="n"/></l>                 <r>koʻpchilik<s n="n"/></r></p></e>
+<e>       <p><l>çukur<s n="n"/></l>                    <r>chuqur<s n="n"/></r></p></e>
+<e>       <p><l>çömlekçilik<s n="n"/></l>              <r>kulolchilik<s n="n"/></r></p></e>
+<e>       <p><l>çözelti<s n="n"/></l>                  <r>eritma<s n="n"/></r></p></e>
+<e>       <p><l>ölçek<s n="n"/></l>                    <r>miqyos<s n="n"/></r></p></e>
+<e>       <p><l>ölçü<s n="n"/></l>                     <r>olcham<s n="n"/></r></p></e>
+<e>       <p><l>ölüm<s n="n"/></l>                     <r>oʻlim<s n="n"/></r></p></e>
+<e>       <p><l>ömür<s n="n"/></l>                     <r>umr<s n="n"/></r></p></e>
+<e>       <p><l>ön<s n="n"/></l>                       <r>old<s n="n"/></r></p></e>
+<e>       <p><l>önem<s n="n"/></l>                     <r>ahamiyat<s n="n"/></r></p></e>
+<e>       <p><l>önlem<s n="n"/></l>                    <r>chora<s n="n"/></r></p></e>
+<e>       <p><l>örme<s n="n"/></l>                     <r>oʻrish<s n="n"/></r></p></e>
+<e>       <p><l>örnek<s n="n"/></l>                    <r>misol<s n="n"/></r></p></e>
+<e>       <p><l>örnek<s n="n"/></l>                    <r>namuna<s n="n"/></r></p></e>
+<e>       <p><l>Özbek<s n="n"/></l>                    <r>oʻzbek<s n="n"/></r></p></e>
+<e>       <p><l>Özbek<s n="n"/></l>                    <r>ozbek<s n="n"/></r></p></e>
+<e>       <p><l>özellik<s n="n"/></l>                  <r>xossa<s n="n"/></r></p></e>
+<e>       <p><l>özgürlük<s n="n"/></l>                 <r>ozodlik<s n="n"/></r></p></e>
+<e>       <p><l>özleştirme<s n="n"/></l>               <r>oʻzlashtirish<s n="n"/></r></p></e>
+<e>       <p><l>ö
```

Comment:
Which one?, I guess you meant the "özleştir" is V-IV, if so I am adding it to the Turkish lexc and the bidix(separate entry) and this "özleştirme" entry is correct I guess, since "geliştirme" is "rivojlantirish"(similar example), right?.